common/ipc-client.c

```diff
@@ -75,7 +75,7 @@ bool ipc_set_recv_timeout(int socketfd, struct timeval tv) {
 		return false;
 	}
 	sway_log(SWAY_DEBUG, "ipc recv timeout set to %ld.%06ld",
-			tv.tv_sec, tv.tv_usec);
+			(long)tv.tv_sec, (long)tv.tv_usec);
```

Comment:
Oops, `.tv_usec` was fine as is.

Reply:
`suseconds_t` is `long` on BSDs/Linux/Solaris, `long long` on Linux [x32](https://en.wikipedia.org/wiki/X32_ABI) and `int` on Linux sparc.
https://git.musl-libc.org/cgit/musl/tree/arch/x32/bits/alltypes.h.in?id=v1.1.22#n23
https://github.com/torvalds/linux/blob/v5.0/arch/sparc/include/uapi/asm/posix_types.h#L19
Cast both `time_t` and `suseconds_t` appears more common.
https://codesearch.debian.net/search?q=printf.*tv_usec